### PR TITLE
Update internal documentation for default process definition

### PR DESCRIPTION
**Pull request by sonchang**

### CODE REVIEW

**Comment by sonchang** [comment]
@ibuildthecloud Just updating the comment.  Haven't looked into the use of '=' (rename?) used in the "transitioning" and "done" definition for the update action yet.  So I just kept it in there.

**Comment by sonchang** [comment]
@cjellick I think this might have fallen on the wayside.  It's just a change to a comment with no code changes.  Do you think you can merge this in?

**Comment by cjellick** [comment]
Ah, yes. This came in when builds were failing all over the place. So, had held off on merging.
LGTM

File: code/iaas/logic/src/main/resources/META-INF/cattle/core-process/spring-core-processes-context.xml [hunk 1]
```diff
@@ -18,15 +18,15 @@
...
 
         The above XML is the equivalent as doing the below, 
-        refer to cattle-process-common/src/main/resources/META-INF/cattle/process/defaults.properties for the definitive list
+        refer to cattle/code/iaas/logic-common/src/main/resources/META-INF/cattle/process/defaults.properties for the definitive list
 
-        TODO: This is out of date
-
-        <process:process name="agent.create" resourceType="agent" start="requested" transitioning="registering" done="inactive" />
+        <process:process name="agent.create" resourceType="agent" start="requested" transitioning="registering" done="inactive" delegate="agent.activate" />
         <process:process name="agent.activate" resourceType="agent" start="inactive" transitioning="activating" done="active" />
```
Comment (cjellick) [inline code/iaas/logic/src/main/resources/META-INF/cattle/core-process/spring-core-processes-context.xml]: Actually, did you miss the registering as a start state for activate?
https://github.com/rancherio/cattle/blob/master/code/iaas/logic-common/src/main/resources/META-INF/cattle/process/defaults.properties#L3

Comment (sonchang) [inline code/iaas/logic/src/main/resources/META-INF/cattle/core-process/spring-core-processes-context.xml]: Good eye!  you're right.  I can update this PR

**Pull request closed without merging**